libs/wingc/src/jsify.rs

```diff
@@ -291,7 +291,9 @@ impl<'a> JSifier<'a> {
 				let expression_type = self.get_expr_type(&expression);
 				let is_preflight_class = expression_type.is_preflight_class();
 
-				let class_type = expression_type.as_class().expect("type to be a class");
+				let class_type = if let Some(class_type) = expression_type.as_class() { class_type } else {
+					return "".to_string();
```

Comment:
Wouldn't this result in invalid JavaScript code? Feels like it should be an error, no?

Reply:
It will be an error: This is for a `new` expressions and our type checker requires that it be a class. We jsify despite that, so this effectively exits early. 
There's no point in adding a new diagnostic since the existing type error is more useful than anything we could say here.